saleor/payment/tasks.py

```diff
@@ -0,0 +1,52 @@
+from celery.utils.log import get_task_logger
+
+from saleor.payment import gateway
+
+from ..celeryconf import app
+from ..plugins.manager import get_plugins_manager
+from .models import Payment
+from .utils import ReleasePaymentException, get_unfinished_payments
+
+task_logger = get_task_logger(__name__)
+
+
+@app.task
+def release_unfinished_payments_task():
+    payments = get_unfinished_payments()
+
+    for payment in payments.iterator():
+        payment.is_active = False
+        payment.save(update_fields=["is_active", "modified"])
```

Comment:
We should change `is_active` value only when `payment` is active.

Reply:
`get_unfinished_payments` filters by `is_active=True`